pages/docs/guidelines/project-closure.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Project Closure Checklists
+section: Guidelines
```

Comment:
Lets move this to "Project Management" section

Section does not exist, we need to create one now